os-certmonger.te

```diff
@@ -11,4 +12,7 @@ read_files_pattern(certmonger_t, puppet_etc_t, puppet_etc_t)
 
 # rhbz#1777368
 container_runtime_domtrans(certmonger_t)
-container_runtime_entrypoint(certmonger_t)
+# Newer version of container-selinux provides this nice interface:
```

Comment:
actually centos-7 also provides similar interface but with name: container_entrypoint instead of container_runtime_entrypoint

Reply:
oh... meh... renaming interface without backward compatibility is wrong X(